CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## v3.11.0
+
+### Added
+
+- Field with path `registration_actions.items[].owner.phone_number` [#83]
+- Field with path `registration_actions.items[].geo.postal_code` [#83]
```

Comment:
`registration_actions.items[].geo.postal_code` is not defined in `fields/default/fields_list.json`. So, we really should add this entry in changelog?